### DISCUSSION TITLE
gh-pages deploy not showing all features

### DISCUSSION THREAD

**Discussion by shaunryan** [squidfunk/mkdocs-material#4567] [Q&A] [answered]
Hi folks

I'm looking at hosting on github pages. When I publish mkdocs with gh-pages -t material I find that a lot of the nice navigation features like the tab menu and custom icons is not working. Is this a know limitation with pages hosting?

**Comment by squidfunk** [comment, marked as answer]
No, everything should work when deploying with gh-pages. We're doing the same for our docs:

https://github.com/squidfunk/mkdocs-material/blob/4caa165b9786d761bf12ae5c0d1f5e6c68a469fe/.github/workflows/documentation.yml#L85

**Reply by shaunryan** [reply]
Just for claity... I was running gh-pages using my local mac as the build machine whilst trying it out and was getting some weird results. I deployed as suggested here using a github workflow and it's working fine. https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions
